src/PiCraft.jl

```diff
@@ -28,7 +28,8 @@ end
 
 module world
     using PiCraft
-    function getBlock(x::Int, y::Int, z::Int)
+    function getBlock(x, y, z)
+        (x, y, z) = (floor(Int64, x), floor(Int64, y), floor(Int64,z))
         res = PiCraft.mc_send("world.getBlock($x,$y,$z)", true)
```

Comment:
While `player` position is a float, a block position is always an Int and must be treated as such to minimize glitch possibilities.